include/ibamr/IBFEMethod.h

```diff
@@ -225,6 +225,8 @@ namespace IBAMR
  *
  *     largest_patch_size
  *     {
+ *         // We recommend using very large values here: large patches
```

Comment:
Should we also increase the min box size?

Reply:
Good question - I would say no because a large minimum patch size makes load balancing harder. I added a bunch of comments explaining the phenomenon.